simple_auth/lib/src/oauth/oauthApi.dart

```diff
@@ -53,49 +53,88 @@ class OAuthApi extends AuthenticatedApi {
 
   OAuthAccount? get currentOauthAccount => currentAccount as OAuthAccount?;
 
+  Future<Account?> handleBrowserRedirect() async {
+    var showPrompt = false;
```

Comment:
Consider making final or inline

Reply:
I'd like to keep the variable for readability, but will make them final.

Reply:
I've  inlined the variables, but added a named parameter.